static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -359,7 +359,7 @@ class CommentBox extends Component {
             const createdDate = new Date(this.getUserData().accountCreatedDate);
 
             if (createdDate > this.options.priorToVerificationDate) {
-                return getUserFromApi().then(response => {
+                return getUserFromApiOrOkta().then(response => {
                     if (
                         response.user.statusFields.userEmailValidated === true
```

Comment:
I think the type of `response` has changed here, and the name doesn't make sense any more. What about changing the variable name to `user` and accessing `statusFields` directly?

```suggestion
                return getUserFromApiOrOkta().then(user => {
                    if (
                        user.statusFields.userEmailValidated
```